tests/functional/preview_and_dev/test_broadcast_flow.py

```diff
@@ -102,7 +102,7 @@ def test_prepare_broadcast_with_new_content(driver):
     assert current_alerts_page.is_text_present_on_page("since today at")
     alert_page_url = current_alerts_page.current_url
 
-    time.sleep(180)
+    time.sleep(10)
```

Comment:
I assume that these sleeps are there to wait until the site has built.  Can we not make a call and determine if the site is currently being built?  Probably out of scope for this change but might be worth creating a tech debt ticket for.

Reply:
If there was a "last modified" field on the index page, we could scrape that timestamp and watch for a change, but this is probably beyond the scope of this ticket.